front/app/components/UI/FormComponents/index.tsx

```diff
@@ -215,6 +217,7 @@ export const FormLabel = memo<
     subtextMessageValues,
     subtextSupportsHtml,
     subtextValue,
+    showContentUploadDisclaimer,
```

Comment:
It's good practice to set a default value for optional props of type boolean (`false` in this case).